typed-racket-lib/typed-racket/base-env/prims-contract.rkt

```diff
@@ -391,9 +393,13 @@
            #,(if (attribute ne)
                  (internal (syntax/loc stx (define-type-alias-internal ty (Opaque pred))))
                  (syntax/loc stx (define-type-alias ty (Opaque pred))))
-           #,(ignore #'(define pred-cnt
-                         (or/c struct-predicate-procedure?/c
-                               (any-wrap-warning/c . c-> . boolean?))))
+           #,(if (attribute unsafe)
+                 (ignore #'(define pred-cnt
```

Comment:
If we wanted that, we can just do `(any/c . ->.  boolean?)`. But I don't think we should add checks for `unsafe-`.